src/structures/user.ts

```diff
@@ -40,7 +40,7 @@ export class User extends Base {
     return `<@${this.id}>`
   }
 
-  avatarURL(format: ImageFormats = 'webp', size: ImageSize = 512, dynamic: boolean = true): string {
+  avatarURL(format: ImageFormats = 'webp', size: ImageSize = 512): string {
```

Comment:
![image](https://user-images.githubusercontent.com/71239005/101116390-5c4bdf00-3628-11eb-9ed1-74778da92da1.png)

As a result of my thoughts, .png seems to be much better than .webp

Reply:
I used the webp extension in discord.js discord.py and discord app, but I changed it because png is more convenient and the official documentation defaults to png.